README.md

```diff
@@ -27,4 +27,4 @@ At the moment there are no tests. Nevertheless, if you run `npm test` this will
 | Fairfax County, VA | [Bicycle routes from Fairfax GIS Data](http://data.fairfaxcountygis.opendata.arcgis.com/datasets/0dacd6f1e697469a81d6f7292a78d30e_16?geometry=-77.32%2C38.826%2C-77.24%2C38.846) | This is a list of "suggested bike routes" (roads under 35mph) and not true bike lanes so we have excluded this data set from the project. |
 | Arlington County, VA | [Bike routes from Arlington County GIS Data](http://gisdata.arlgis.opendata.arcgis.com/datasets/af497e2747104622ac74f4457b3fb73f_4?geometry=-77.295%2C38.81%2C-76.87%2C38.89) | We filtered by "Off Street Trail" and "Marked Route". |
 | Alexandria City, VA | [Bike trails from Alexandria Open GIS Data Portal](http://data.alexgis.opendata.arcgis.com/datasets/685dfe61f1aa477f8cbd21dceb5ba9b5_0) | "Collection of trails within the City of Alexandria, Virginia. Shows trail features categorized by on street, off street, and sharrow (bike lane marked)." |
-| Prince George's County, MD | [Master Plan Trails from PGAtlas](http://www.pgatlas.com/?mapID=3b60528c-dddf-4958-9107-d4c458c295c1) & [Bike Trails Layer from MWCOG](http://gis.mwcog.org/wa/rest/services/RTDC/PGCTrails/MapServer/0) | Data was not easily extractable from the open data portal and as a result was not included in this project. |
+| Prince George's County, MD | [Master Plan Trails from PG Planning Open Data Portal](http://gisdata.pgplanning.org/opendata/) | Filtered by FACILITY_S = "Existing" AND "FACILITY_T = "Bike Lane" OR "FACILITY_T" = "Hard Surface Trail" so as to include only facilities that have been built (not just planned) and only bike lanes and paved trails. |
```

Comment:
Can we add the URL to the ShapeFile to the Master Plan Trail? I had a hard time finding it.

http://gisdata.pgplanning.org/opendata/downloadzip.asp?FileName=/data/ShapeFile/Master_Plan_Trail_Ln.zip